wsagent/che-core-api-project-shared/src/main/java/org/eclipse/che/api/project/shared/dto/event/GitBranchCheckoutEventDto.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.project.shared.dto.event;
+
+import com.google.common.annotations.Beta;
+
+import org.eclipse.che.dto.shared.DTO;
+
+/**
+ * To transfer branch name after git checkout operation
```

Comment:
Git checkout can be performed not only with branch name but also commitId, and other links (besides branch name) to commitId. 
Is this class just an example? Will it handle other checkout variants later?